--- origsrc/gnome-todo-3.22.1/src/Makefile.am	2016-12-19 18:22:57.000000000 -0600
+++ src/gnome-todo-3.22.1/src/Makefile.am	2017-03-07 12:00:52.065470000 -0600
@@ -217,10 +217,10 @@ introspection_sources = \
 	gtd-types.h \
 	$(NULL)
 
-Gtd-1.0.gir: gnome-todo
+Gtd-1.0.gir: gnome-todo$(EXEEXT)
 Gtd_1_0_gir_INCLUDES = Gio-2.0 GObject-2.0 Gtk-3.0
 Gtd_1_0_gir_CFLAGS = $(gnome_todo_CFLAGS)
-Gtd_1_0_gir_PROGRAM = $(builddir)/gnome-todo
+Gtd_1_0_gir_PROGRAM = $(builddir)/gnome-todo$(EXEEXT)
 Gtd_1_0_gir_SCANNERFLAGS = \
 	-n Gtd \
 	--identifier-prefix Gtd  \
